packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -279,6 +279,11 @@ class LitDevSearch extends LitElement {
    * refreshing the UI.
    */
   private navigate(url: string) {
+    // TODO(https://github.com/lit/lit.dev/issues/423) - Land Lit.dev search
+    // It is safe to append this directly as no url included in the searchIndex
+    // will have any query params.
+    url += `?mods=search`;
```

Comment:
This won't quite work if there's an anchor in the URL, since it will create e.g. `/docs/api/custom-directives/#isTemplateResult?mods=search` with the parameter in the wrong place. Suggest using the `URL` API and calling `searchParams.set`, like here: https://github.com/lit/lit.dev/blob/main/packages/lit-dev-content/src/global/mods.ts#L18